apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,50 @@
 import { Component, OnInit } from '@angular/core';
+import { CityStore } from '../../data-access/city.store';
+import {
+  FakeHttpService,
+  randomCity,
+} from '../../data-access/fake-http.service';
+import { City } from '../../model/city.model';
+import { CardComponent } from '../../ui/card/card.component';
+import { CardListInterface } from '../../interfaces/card-list.interface';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
   standalone: true,
-  imports: [],
+  imports: [CardComponent],
+  template: `<app-card
+    [list]="cities"
+    customClass="bg-light-blue"
+    (addButtonClicked)="handleAdd()"
+    (deleteButtonClicked)="handleDelete($event)">
+    <img
+      role="card-image"
+      src="assets/img/cityscape-sunset-vector.jpg"
+      width="200px" />
+  </app-card>`,
+  styles: [
+    `
+      :host {
+        --bg-light-blue: rgba(0, 0, 250, 0.1);
+      }
+    `,
+  ],
 })
-export class CityCardComponent implements OnInit {
-  constructor() {}
+export class CityCardComponent implements OnInit, CardListInterface {
+  cities: City[] = [];
+  constructor(private http: FakeHttpService, private store: CityStore) {}
 
-  ngOnInit(): void {}
+  ngOnInit(): void {
+    this.http.fetchCities$.subscribe((s) => this.store.addAll(s));
+
+    this.store.cities$.subscribe((s) => (this.cities = s));
```

Comment:
you can use an asyncPipe to avoid imperative coding

Reply:
changed to asyncPipe, thanks for the feedback